lib/Elevate/PkgMgr/YUM.pm

```diff
@@ -254,13 +254,16 @@ sub update_with_options ( $self, $options, $pkgs ) {
     return;
 }
 
-sub update_allow_erasing ( $self, @args ) {
+sub update_allow_erasing ( $self, @additional_args ) {
     my $pkgmgr = $self->_pkgmgr();
 
-    my @additional_args = scalar @args ? @args : '';
-
-    $self->ssystem( $pkgmgr, '-y', '--allowerasing', @additional_args, 'update' );
+    my @args = (
+        '-y',
+        '--allowerasing',
+    );
 
+    push @args, @additional_args;
+    $self->ssystem_and_die( $pkgmgr, @args, 'update' );
```

Comment:
so the only change is `ssystem_and_die` ?

Reply:
There was the potential for `@args` to be `''` when no args were passed into `update_allow_erasing`.  For whatever reason, the empty string was causing the call to dnf to fail so I just made it to where that empty string could never be present.